diff -Naur pyOpenSSL-16.2.0/src/OpenSSL/SSL.py pyOpenSSL-16.2.0.patched/src/OpenSSL/SSL.py
--- pyOpenSSL-16.2.0/src/OpenSSL/SSL.py	2016-10-15 09:15:43.000000000 -0400
+++ pyOpenSSL-16.2.0.patched/src/OpenSSL/SSL.py	2016-12-18 09:28:14.885819156 -0500
@@ -1247,6 +1247,8 @@
 
         if isinstance(buf, _memoryview):
             buf = buf.tobytes()
+        elif isinstance(buf, basestring):
+            buf = bytes(buf)
         if isinstance(buf, _buffer):
             buf = str(buf)
         if not isinstance(buf, bytes):
@@ -1272,6 +1274,8 @@
 
         if isinstance(buf, _memoryview):
             buf = buf.tobytes()
+        elif isinstance(buf, basestring):
+            buf = bytes(buf)
         if isinstance(buf, _buffer):
             buf = str(buf)
         if not isinstance(buf, bytes):
